build/vs/pjproject-vs14-common-config.props

```diff
@@ -18,20 +18,20 @@
   <PropertyGroup>    
     <API_Family Condition="'$(API_Family)'==''">WinDesktop</API_Family>
     <PreprocessorDef></PreprocessorDef>
-    <DefaultToolset>v140</DefaultToolset>
```

Comment:
Ok, file restored

Reply:
Still the same. I think you can safely exclude it from the patch.

Reply:
created PR #4212
Now github sees only some changes in this file, but not in other files...
And Fork (the git client I use) shows only changed lines.
The only idea I have is that VS likes Windows CR/LF line endings and can automatically switch to them.

Reply:
Your editorconfig is probably the reason for all these changes.

I understand you may prefer your local settings, but since it's a public project, it's better to use the default config so everybody adopts the same setting.